### PR TITLE
release(v0.11.0-alpha.1): prepare release

### PR DESCRIPTION
**Pull request by smira**
This is the official v0.11.0-alpha.1 release.

Signed-off-by: Andrey Smirnov <smirnov.andrey@gmail.com>

---
This change is [Reviewable](https://reviewable.io/reviews/talos-systems/talos/3781)

### CODE REVIEW

**Review by andrewrynhard — DISMISSED** [review]
Exciting!

**Review by rsmitty — DISMISSED** [review]

File: hack/release.toml [hunk 1]
```diff
@@ -54,6 +54,35 @@ Users can use a modified `controlplane.yaml` with the machine type set to
...
+Default `talosconfig` has `os:admin` role embedded in the certificate so that all the APIs are available.
+Certificates with reduced set of roles can be created with `talosctl config new` command.
+
+When upgrading from Talos 0.10, RBAC is not enabled by default. Before enabling RBAC, generate `talosconfig` with
+`os:admin` role first to make sure that administrator still have access to the cluster when RBAC is enabled.
+
+List of available roles:
+
+* `os:admin` role enables every Talos API
+* `os:reader` role limits access to read-only APIs which do not return sensitive informtation
```
Comment (Unix4ever) [inline hack/release.toml:81]:
```suggestion
* `os:reader` role limits access to read-only APIs which do not return sensitive information
```

File: hack/release.toml [hunk 2]
```diff
@@ -54,6 +54,35 @@ Users can use a modified `controlplane.yaml` with the machine type set to
...
+
+    [notes.rbac]
+        title = "Talos API RBAC"
+        description = """\
+Limited RBAC support in Talos API is now enabled by default for Talos 0.11.
+Default `talosconfig` has `os:admin` role embedded in the certificate so that all the APIs are available.
+Certificates with reduced set of roles can be created with `talosctl config new` command.
+
+When upgrading from Talos 0.10, RBAC is not enabled by default. Before enabling RBAC, generate `talosconfig` with
+`os:admin` role first to make sure that administrator still have access to the cluster when RBAC is enabled.
```
Comment (Unix4ever) [inline hack/release.toml:76]:
```suggestion
`os:admin` role first to make sure that administrator still has access to the cluster when RBAC is enabled.
```

**Review by Unix4ever — APPROVED** [review]

File: hack/release.toml (hunk 1 above)
Comment (smira) [reply hack/release.toml:81]: :facepalm:

File: hack/release.toml (hunk 2 above)
Comment (smira) [reply hack/release.toml:76]: :facepalm:

**Pull request merged**